src/Writers/HTMLWriter.jl

```diff
@@ -478,22 +478,23 @@ function render_topbar(ctx, navnode)
 end
 
 function generate_version_file(dir::AbstractString)
-    named_folders = []
-    tag_folders = []
-    for each in readdir(dir)
-        each in ("stable", "latest")         ? push!(named_folders,   each) :
-        occursin(Base.VERSION_REGEX, each)   ? push!(tag_folders,     each) : nothing
+    all_folders = readdir(dir)
+    folders = []
+
+    for each in all_folders
+        occursin(Base.VERSION_REGEX, each) && push!(folders, each)
     end
-    # put stable before latest
-    sort!(named_folders, rev = true)
     # sort tags by version number
-    sort!(tag_folders, lt = (x, y) -> VersionNumber(x) < VersionNumber(y), rev = true)
+    sort!(folders, lt = (x, y) -> VersionNumber(x) < VersionNumber(y), rev = true)
+
+    # include stable first, and dev at the bottom
+    "stable" in all_folders && pushfirst!(folders, "stable")
+    "dev" in all_folders && push!(folders, "dev")
```

Comment:
I would keep "dev" at the top, since that's usually the one you want. At least as long as we have all the patch versions in the selector.